Jenkinsfile.rh.release

```diff
@@ -24,6 +24,46 @@ properties([
   ]),
 ])
 
+String version = ''
+String checksum = ''
+
+dockerizedBuildPipeline(
+  deployBranch: 'main',
+  prepare: {
```

Comment:
There is a prepare stage already I think you can put this stuff there instead of declaring another pipeline

Reply:
Commit: 8912bac3a158efe5943e9c752169ca611c7d4cc5